<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupDialog</class>
 <widget class="QDialog" name="SignupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>球员</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>三分球个数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>球队</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>比赛日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>篮板</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditThree">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRebound">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>扣篮</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>得分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>抢断</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditSteal">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditDunk">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditScore">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboTeamname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>金</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>木</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>水</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>火</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>土</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>SignupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SignupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
